hasura/graphql-engine
# close websocket connection on JWT expiry (fix #578)

### Description


Close WebSocket connection while subscriptions on JWT expiry.

### Affected components 


- Server
- Tests

### Related Issues



close #578 

### Solution and Design



Run an async thread on websocket connection accept which terminates on JWT expiry.

### Steps to test and verify


Perform a subscription query with JWT setup. The server should close connection on JWT expiry.

### Limitations, known bugs & workarounds
